src/components/Select/Select.js

```diff
@@ -108,12 +131,16 @@ const SelectContainer = styled('div')`
 const SelectElement = styled('select')`
   ${selectBaseStyles};
   ${selectInvalidStyles};
+  ${selectPrefixStyles};
 `;
 
 const Icon = styled(ArrowsIcon)`
   ${iconBaseStyles};
 `;
 
+const renderFixComponent = (className, renderFn) =>
+  (renderFn && renderFn({ className })) || null;
```

Comment:
**Just a nitpick** 
There is no need to do `falsy || null` here cause the `defaultProps` is `null` and React handles "Falsy" values in the same way accordingly with their docs 😄
https://reactjs.org/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored

Reply:
👍